pipeline/beam_tables.py

```diff
@@ -873,6 +878,9 @@ def __init__(self, project: str, bucket: str, staging_location: str,
     # Maxmind is also too big to pass around
     self.maxmind_class = maxmind_class
     self.maxmind_bucket_folder = maxmind_bucket_folder
+    # DBIP is also too big to pass around
+    self.dbip_class = dbip_class
```

Comment:
Yeah I agree, I'd like to do something like

pipeline.metadataIPMetadataChooser
which contains instances all the caida/dbip/maxmind modules and the business logic to select different values between them and augment a row.

And then a sort of
ProductionMetadataValues class
which would hold all these values and get passed around as one thing instead of 6.

But yes I'd like to do it in a separate pr. Added a TODO.